vuepress/docs/next/docs/reference/deployment-structure.md

```diff
@@ -58,13 +62,13 @@ container pod.
 #### Customization
 The _entando-de-app_ is very likely to be customized as part of an Entando implementation. This image can be 
```

Comment:
backticks instead of underscore and italics

reads awkwardly, like it's customized out of the box. rephrase to how the user will want to customize?